slate/build.boot

```diff
@@ -19,15 +23,15 @@
 (deftask package  []
   (comp
    (download :url (str "https://unpkg.com/slate@" +lib-version+  "/dist/slate.js")
-             :checksum "6E87C8294DE916E9496A582079B43CBF")
+             :checksum "eae6213c6fe071154a6ca9868b29f688")
    (download :url (str "https://unpkg.com/slate@" +lib-version+  "/dist/slate.min.js")
-             :checksum "9CB6F87CA7ABC89344476F6235EBC2D5")
+             :checksum "f50c3828c0ca9040d9047612aa8745bf")
    (sift :move {#"^slate.js$"
                 "cljsjs/slate/development/slate.inc.js"
                 #"^slate.min.js"
                 "cljsjs/slate/production/slate.min.inc.js"})
    (sift :include #{#"^cljsjs"})
    (deps-cljs :name "cljsjs.slate"
-              :requires ["cljsjs.react" "cljsjs.react.dom" "cljsjs.immutable"])
+              :requires ["cljsjs.react" "cljsjs.immutable"])
```

Comment:
Does this package still depend on React? If foreign-lib requires react, the package should also have dependency on cljsjs/react package.